btrdb/stream.py

```diff
@@ -899,6 +899,7 @@ def arrow_values(
         self,
         start,
         end,
+        sorted: bool = False,
```

Comment:
Raw values returns unsorted data?

Reply:
I am unsure if that is the case or not. I dont recall ever running into that, but since we have the option to sort with pyarrow, i decided to add in that option.

Reply:
Based on discussions, i am going to remove the sort ability for the raw values since it is very unlikely that raw points are returned in non-sorted order.